src/main/webapp/resources/js/pages/analyses/analysis-outputs-table.js

```diff
@@ -1,4 +1,4 @@
-import { Grid } from "ag-grid-community/main";
```

Comment:
Add file comment

Reply:
Added in 807aabb78585b3f6477f9212bbf800ad496855cc